libs/vscode/miranum-vscode/src/lib/fileSystem/Reader.ts

```diff
@@ -1,7 +1,12 @@
 import * as vscode from "vscode";
-import { Uri } from "vscode";
-import { FolderContent, WorkspaceFolder } from "../types";
-import { Logger } from "./Logger";
+import { Logger } from "../Logger";
+import { FolderContent } from "@miranum-ide/vscode/miranum-vscode-webview";
```

Comment:
@lmoesle I don't realy like this dependency. But I do need the data structure in the webview to extract it correctly. Maybe you have a better solution.